### DISCUSSION TITLE
Contribution Guidelines

### DISCUSSION THREAD

**Discussion by mweisgut** [hyrise/hyrise#2494] [General]
How do we deal with coding aspects that are not covered in the contribution guidelines?

Options
- choose one of the widely known style guides and use their solution if one aspect is not covered by Hyrise's style guide
- adopt as much as possible from the core guidelines https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines

**Comment by Bouncner** [comment]
Do we have good examples here?

I think it is hard to find "the one answer" for this topic. I personally look at large style guides and try to make up my mind. There are several things I don't like in the LLVM style guide, but much of it sounds and looks very reasonable to me. But I wouldn't go so far to select a single style guide.

**Reply by Bouncner** [reply]
@mweisgut : pinging here, since I don't know if you got a notification.

**Reply by mweisgut** [reply]
I don't have an exact example. This discussion came up every now and then, so we decided to keep it in the discussion section. I do not have a concrete example.
